vcluster/configure/vcluster-yaml/sync/to-host/advanced/custom-resources.mdx

```diff
@@ -41,20 +44,38 @@ sync:
         enabled: true
 ```
 
+## Versioned example
+
+To use a specific version of a CustomResourceDefinition (CRD), specify the version before the resource type in the `apiVersion` field, using the `/` separator. For example, to use version `v1` of the `certificates.cert-manager.io` CRD:
+
+```yaml
+sync:
+  toHost:
+    customResources:
+      certificates.cert-manager.io/v1:
+        enabled: true
+```
+
+:::info Unique CRD versions
+vCluster supports syncing only one version of a custom resource. If multiple versions are specified, the virtual cluster fails to start.
+
+If an upgrade introduces a version that is not yet synced, vCluster modifies the CRD inside the virtual cluster to include the new version.
+:::
```

Comment:
I am not totally clear on what this info blurb is trying to explain.

Are we saying that during a vCluster upgrade, if a new version is added to the yaml and it's not synced, then it will sync that version now?

What happens if I go from no version (uses storage version) and then upgrade my vcluster to update the yaml to specify a version?

Reply:
> Are we saying that during a vCluster upgrade, if a new version is added to the yaml and it's not synced, then it will sync that version now?

Yes.

> What happens if I go from no version (uses storage version) and then upgrade my vcluster to update the yaml to specify a version?

In this case, if the chosen version is the same as the `storage` version then the CRD doesn't need to be updated.
A new version will be added to the CRD only if it's needed and not in the vCluster's CRD.

Please note that the same will happen if we start with an explicit version like `clusterissuers.cert-manager.io/v1alpha1` then upgrade to `clusterissuers.cert-manager.io` (without a version) if the `storage` version is set to `v1`. In this case `v1` will be added to the vCluster's CRD.

Reply:
Added